app/script/telemetry/calling/CallLogger.js

```diff
@@ -0,0 +1,186 @@
+      MESSAGE_LOG_LENGTH: 10000,
+      OBFUSCATION_TRUNCATE_TO: 2,
+    };
+  }
+
+  obfuscate(string) {
+    //return CryptoJS.SHA256(`${string}${this.currentWeek}`)
+    //  .toString()
+    return string.substr(0, CallLogger.CONFIG.OBFUSCATION_TRUNCATE_TO);
+  }
+
+  obfuscateIp(ip) {
+    if (CallLogger.CONFIG.IPV4.test(ip)) {
+      // IPv4
+      ip = ip.split('.');
+      ip[ip.length - 1] = '0';
+      return ip.join('.');
+    } else if (CallLogger.CONFIG.IPV6.test(ip)) {
+      // IPv6
+      ip = ip.split(':').slice(0, 3);
+      return [...ip, '0000', '0000', '0000', '0000'].join(':');
+    }
+
+    return 'Unknown';
+  }
+
+  /*fakeIpGenerator(seed, type) {
+    if (type === 4) {
+      // Extend the 4 bytes seed to 16 bytes using XOR encryption with static keys
+      const originalSeedLength = seed.length;
+      const keys = [21, 15, 7];
+      for (let i = 0; i < originalSeedLength; ++i) {
+        for (const key of keys) {
+          seed += String.fromCharCode(key ^ seed.charCodeAt(i));
+        }
+      }
+
+      // Generate a fake IP from that seed
+      const fakeIp = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+      for (let i = 0; i < seed.length; i++) {
+        fakeIp[i % fakeIp.length] += seed.charCodeAt(i);
+      }
+      return fakeIp
+        .map(v => {
+          return (v % 10).toString(10);
+        })
+        .join('')
+        .match(/.{1,3}/g)
+        .join('.');
+    } else if (type === 6) {
+      const fakeIp = CryptoJS.MD5(seed).toString();
+      return fakeIp.match(/.{1,4}/g).join(':');
+    }
+  }
+
+  removeBytesFromIp(ip) {
+    let charactersToKeep;
+    let type;
+
+    ip = ip.replace(/[\.\:]/g, '');
+
+    if (ip.length > 12) {
+      // IPv6 - Without colons the maximum is 32 characters
+      type = 6;
+      charactersToKeep = 12;
+    } else {
+      // IPv4 - Without dots the maximum is 12 characters
+      type = 4;
+      charactersToKeep = 4;
+      if (ip.length > 7) {
+        charactersToKeep = 6;
+      } else if (ip.length > 6) {
+        charactersToKeep = 5;
+      }
+    }
+
+    const mid = Math.round((ip.length + 1) / 2);
+
+    return {
+      slicedIp: ip.slice(mid - charactersToKeep / 2, mid + charactersToKeep / 2),
+      type,
+    };
+  }*/
+
+  obfuscateSdp(sdpMessage, conversationId) {
+    const decodedSdpMessage = sdpTransform.parse(sdpMessage);
+
+    for (const index in decodedSdpMessage.media) {
+      // Remove fingerprint
+      if (
+        typeof decodedSdpMessage.media[index].fingerprint !== 'undefined' &&
+        typeof decodedSdpMessage.media[index].fingerprint.hash !== 'undefined'
+      ) {
+        decodedSdpMessage.media[index].fingerprint.hash =
+          'XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX';
```

Comment:
I suppose the fake data can be shortened

Reply:
This is the size of the fingerprint. We must keep the same size for readability. Having a .repeat() is possible but would make an additional function call so better keeping it like this.

Reply:
Why do we need to keep it for readability? You can change it to "OBFUSCATED" and it will be obvious what you did as wel. Why should we print 26 couples of Xs here?

Reply:
So it looks like a fingerprint and it's easier to read as "X" are used for removed data. I moved those things to the config file so calls are only done once: https://github.com/wireapp/wire-webapp/pull/2797/files#diff-2d0c04a493e7ca82351e008e3e9ebd30R93 and used `Array.from({length: 32}).fill('XX').join(':')` for this specific line.